cmd/codegen/main.go

```diff
@@ -71,6 +83,35 @@ func ClientGen(cmd *cobra.Command, args []string) error {
 	return Generate(ctx, cfg, dag)
 }
 
+func Introspect(cmd *cobra.Command, args []string) error {
```

Comment:
That's nice!!

Reply:
Yup 🙂 At some point it can come from the codegen module, but this was a quicker fix for now and can be reused in other SDKs.

Reply:
I have plan to use the introspection from that module. But after see this PR, I will wait until this PR merge then. 😂